partiql/tests/snapshots/ion__ion_structs.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: partiql/tests/ion.rs
+expression: "result.to_pretty_string(25).expect(\"pretty\")"
+---
+<<
+  {
+    'data': `annot1::(1 hi::2)`
+  },
+  {
+    'data': `annot2::{data: 3}`
+  }
```

Comment:
Nice to see this work. Backticks move around `x.data` and it's still kept as Ion data without eagerly converting to a PartiQL value.